charts/keycloakx/ci/h2-values.yaml

```diff
@@ -1,7 +1,7 @@
 command:
   - "/opt/keycloak/bin/kc.sh"
   - "--verbose"
-  - "start-dev"
+  - "start"
```

Comment:
I think the test is failing because of booting in production mode.
It is necessary to run in production mode for testing?

Reply:
I just created test that launch in production mode and passes.

https://github.com/cocor-au-lait/keycloak-helm-charts/pull/3/files#diff-a2787f787ce9c889ac6c604daa1c10fce6d02e7478333378f82bf541863a0b70R22-R25

To pass the test, I've disabled strict hostname and https.